mozjexl/lib/idsintree.js

```diff
@@ -0,0 +1,37 @@
+
+expression = process.argv[2] || 'normandy.clientId=="abced" && channel=="nightly"';
+console.log(`
+
+  EXPRESSION TO PARSE
+
+  ${expression}
+
+`)
+
+var Evaluator = require('./evaluator/Evaluator'),
+  Lexer = require('./Lexer'),
+  Parser = require('./parser/Parser'),
+  defaultGrammar = require('./grammar').elements;
+
+P = new Parser(defaultGrammar);
+P.addTokens(new Lexer(defaultGrammar).tokenize(expression))
```

Comment:
I think mozjexl could expose a nicer API than this. We could probably expose a tree walking API directly.